test/test-build.js

```diff
@@ -39,6 +39,18 @@ function doTests(transpiler) {
     });
   });
 
+  test('Static string build', function () {
+    builder.reset();
+    builder.config({ transpiler: transpiler });
+    return builder.bundle('foo.js', {
+      fetch: function (data) {
+        if (data.name.indexOf('foo.js') !== -1) {
+          return fs.readFileSync('test/fixtures/test-tree/amd-1.js', 'utf8');
+        }
+      }
```

Comment:
Thanks! Do you think it is good practice to override the loader's functions like that? It might make it less predictable what the fetch function does in other areas of the framework, making it less modular.

What do you think about adding a fallthrough case as well?
```javascript
function (load) {
  var ret = builder.fetch(load);
  if (!ret) ret = loaderFetch(load);
  return ret;
}
```

This way it will fall through to using the default loader if you do not return anything from your `fetch` function.